spanner/ddl/ast.go

```diff
@@ -42,13 +42,15 @@ const (
 	String string = "STRING"
 	// Timestamp represent TIMESTAMP type.
 	Timestamp string = "TIMESTAMP"
+	// Numeric represent NUMERIC type.
+	Numeric string = "NUMERIC"
 	// MaxLength is a sentinel for Type's Len field, representing the MAX value.
 	MaxLength = math.MaxInt64
 )
 
 // Type represents the type of a column.
 //     type:
-//        { BOOL | INT64 | FLOAT64 | STRING( length ) | BYTES( length ) | DATE | TIMESTAMP }
+//        { BOOL | INT64 | FLOAT64 | STRING( length ) | BYTES( length ) | DATE | TIMESTAMP | NUMERIC }
```

Comment:
We should probably update postgres and mysql with NUMERIC.
Probably best to track that with an issue?

Reply:
I created an issue #87 for this.